javascript/src/main/pages/Statistics/SingleCourseSearch.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useState } from "react";
+import { Jumbotron, Container } from "react-bootstrap";
+import { fetchSingleCourseSearch } from "main/services/statisticsService";
+import SingleCourseSectionForm from "main/components/Statistics/SingleCourseSectionForm";
+import SingleCourseSectionTable from "main/components/Statistics/SingleCourseSectionTable";
+
+const SingleCourseSearch = () => {
+    const [tableVisibility, setTableVisibility] = useState(false);
+    const [tableData,setTableData] = useState([]);
+
+    const setJsonTableData = (json) => {
+        const keys = Object.keys(json);
+        let arr = [];
+
+        keys.forEach(function (item) {
+            var temp = {"professor":item, "professorCount":json[item]}
+            arr.push(temp);
```

Comment:
Since this variable is only pushed to `arr` we should probably directly just push it onto the array, with `arr.push({"professor":item, "professorCount":json[item]})`.

Secondly, we want to be consistent with using var and let in javascript, as they they have subtle [differences](https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var).

I also feel this may be too nit-picky, please let me know your thoughts.

Reply:
We removed the temp variable